ngx_http_zip_parsers.c

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
@evanmiller: how do you generate this file? No matter which options I pass to ragel, it uses different formatting, and even different logic. Or do you re-format the code manually?

Reply:
Could be an older or newer version of Ragel. What version are you using?

Reply:
I have 7.0.0.12 (that's what Fedora has in its repo). What about you?

Reply:
Ah, yeah Ragel 7 uses a completely new architecture based on [Colm](http://www.colm.net/open-source/colm/). I've been using 6.x. I don't mind either way as long as we standardize.

Reply:
Rebased it.